SUILesson1/ContentView.swift

```diff
@@ -7,15 +7,108 @@
 
 import SwiftUI
 
+/// Представление с контентом экрана мини приложения
 struct ContentView: View {
+
+    private enum Constants {
+        static let defaultLabelText = "Текст лейбла для замены"
+        static let showButtonText = "Показать"
+        static let alertExampleText = "Пример алерта"
+        static let sayAText = "Сказать А"
+        static let sayAReplicText = "Аааааааа"
+        static let sayBText = "Сказать Б"
+        static let sayBReplicText = "Б-б-б"
+        static let showActionSheetText = "Показать шторку"
+        static let actionsText = "Действия"
+        static let actionsExemples = "Пример ActionSheet"
+        static let defaultEmptyText = ""
+    }
+
+    // MARK: - private properties
+
+    @State private var isErrorFirst = false
+    @State private var isErrorSecond = false
+    @State private var isPresented = false
+    @State private var labelText = Constants.defaultLabelText
+
+    // MARK: - public properties
+
     var body: some View {
         VStack {
-            Image(systemName: "globe")
-                .imageScale(.large)
-                .foregroundColor(.accentColor)
-            Text("Hello, world!")
+            Form {
+                Section {
+                    Text(labelText)
+                }
+
+                Section {
+                    makeFirstButton()
+                }
+
+                Section {
+                    makeButtonWithDoubleButtonAlert()
+                }
+
+                Section {
+                    makeActionSheertButton()
+                }
+
+                Section {
+                    makeBottomActionSheerButton()
+                }
+            }
         }
-        .padding()
+    }
+
+    private func makeFirstButton() -> some View {
```

Comment:
```suggestion
    private var makeFirstButton: some View {
```
 если нет параметров, то это вычисляемое свойство

Reply:
ниже также